src/staticData.js

```diff
@@ -2,104 +2,130 @@ const disabilityList = [
   {
     key: "x",
     text: "wybierz przyczynę...",
+    textGenetive: "",
+    textAdjective: "",
     value: "",
     multiple: false,
     disable: false
   },
   {
     key: "0",
     text: "niepełnosprawność intelektualna w stopniu lekkim",
+    textGenetive: "niepełnosprawność intelektualną w stopniu lekkim",
+    textAdjective: "niepełnosprawne intelektualnie w stopniu lekkim",
     value: "LEKKIE",
     multiple: true,
     disable: false
   },
   {
     key: "1",
     text: "niepełnosprawność intelektualna w stopniu umiarkowanym",
+    textGenetive: "niepełnosprawność intelektualną w stopniu umiarkowanym",
+    textAdjective: "niepełnosprawne intelektualnie w stopniu umiarkowanym",
     value: "UMIARKOWANE",
     multiple: true,
     disable: false
   },
   {
     key: "2",
     text: "niepełnosprawność intelektualna w stopniu znacznym",
+    textGenetive: "niepełnosprawność intelektualną w stopniu znacznym",
+    textAdjective: "niepełnosprawne intelektualnie w stopniu znacznym",
     value: "ZNACZNE",
     multiple: true,
     disable: false
   },
   {
     key: "3",
     text: "niepełnosprawność intelektualna w stopniu głębokim",
+    textGenetive: "niepełnosprawność intelektualną w stopniu głębokim",
+    textAdjective: "niepełnosprawne intelektualnie w stopniu głębokim",
     value: "GŁĘBOKIE",
     multiple: false,
     disable: false
   },
   {
     key: "4",
-    text: "niepełnosprawne ruchowo, w tym z afazją",
+    text: "niepełnosprawność ruchowa, w tym z afazją",
+    textGenetive: "niepełnosprawność ruchową, w tym z afazją",
+    textAdjective: "niepełnosprawne ruchowo, w tym z afazją",
     value: "RUCHOWA",
     multiple: true,
     disable: false
   },
   {
     key: "5",
     text: "słabosłyszenie",
+    textGenetive: "słabosłyszenie",
+    textAdjective: "słabosłyszące",
     value: "SLABOSLYSZACE",
     multiple: true,
     disable: false
   },
   {
     key: "6",
     text: "niesłyszenie",
+    textGenetive: "niesłyszenie",
+    textAdjective: "niesłyszące",
     value: "NIESLYSZACE",
     multiple: true,
     disable: false
   },
   {
     key: "7",
     text: "słabowidzenie",
+    textGenetive: "słabowidzenie",
+    textAdjective: "słabowidzące",
     value: "SLABOWIDZACE",
     multiple: true,
     disable: false
   },
   {
     key: "8",
     text: "niewidzenie",
+    textGenetive: "niewidzenie",
+    textAdjective: "niewidzące",
     value: "NIEWIDZACE",
     multiple: true,
     disable: false
   },
   {
     key: "9",
     text: "autyzm w tym zespół Aspergera",
+    textGenetive: "autyzm w tym zespół Aspergera",
+    textAdjective: "z autyzmem, w tym z zespołem Aspergera",
     value: "AUTYZM",
     multiple: true,
     disable: false
   },
   {
     key: "10",
     text: "stan zdrowia uniemożliwiający uczęszczanie do szkoły",
+    textGenetive: "stan zdrowia uniemożliwiający uczęszczanie do szkoły",
     value: "UNIEMOZLIWIAJACY",
     multiple: false,
     disable: false
```

Comment:
Here you don't add `textAdjective` and I know some values from these objects won't be used but I strongly recommend the consistency of objects in the array even if some values will be empty. (the same for key: 11,12 and 13)